.github/workflows/build-test.yaml

```diff
@@ -2567,21 +2571,16 @@ jobs:
             exit 1
           fi
 
-          if [ "$(kubectl get all -n postgres-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "postgres-test namespace still contains resources\n\n"
-            kubectl get all -n postgres-test
-            exit 1
-          fi
 
-          if [ "$(kubectl get all -n redis-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "redis-test namespace still contains resources\n\n"
-            kubectl get all -n redis-test
+          if [ "$(kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
+            printf "%s namespace still contains resources\n\n" "$ns"
+            kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG"
```

Comment:
this confirms that there are no resources left for the app

Reply:
Actually, I think this still fails to see the helm release secret with type `helm.sh/release.v1`. Maybe we need to add a check for: `kubectl get secrets --field-selector type=helm.sh/release.v1 -A` ?